app/controllers/spirits_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
     if @spirit.save
       flash[:notice] = "Spirit was created successfully"
       redirect_to spirits_path
-    else render :new, status: :unprocesable_entity
+    else render :new, status: :unprocessable_entity
```

Comment:
Let's put the `render` on its own line. This is valid as-is but typically in Ruby we keep the `else` line empty of anything else.

That reminds me...we should set up rubocop to catch things like this.

Reply:
Done! And yes, rubocop soon.